### DISCUSSION TITLE
RealtionManager inside a tab

### DISCUSSION THREAD

**Discussion by medfrk** [filamentphp/filament#10542] [Help]
I have  this scenario i want to show a relationManager not on the bottom of my form inside a tab  of my form. 
<img width="1778" alt="Screenshot 2023-12-28 at 17 35 54" src="https://github.com/filamentphp/filament/assets/2337817/7324888f-2d91-4cf9-ac57-07d10a90f7a5">

**Comment by leandrocfe** [comment]
What about this?
https://filamentphp.com/docs/3.x/panels/resources/relation-managers#combining-the-relation-manager-tabs-with-the-form

**Reply by medfrk** [reply]
Thank you for your response. I have already tried this solution, but it's not exactly what I'm looking for. I hope that Filament will offer a way to position the relation manager where desired, not just at the bottom.
